lib/fileProcessor.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+class FileProcessor {
+  constructor(store, files) {
```

Comment:
Is this change on your local machine? I don't see a logger at line 14 of gitHubProcessor in develop.